scrapy/core/scheduler.py

```diff
@@ -28,7 +28,7 @@ def from_crawler(cls, crawler):
         dupefilter = dupefilter_cls.from_settings(settings)
         dqclass = load_object(settings['SCHEDULER_DISK_QUEUE'])
         mqclass = load_object(settings['SCHEDULER_MEMORY_QUEUE'])
-        logunser = settings.getbool('LOG_UNSERIALIZABLE_REQUESTS')
```

Comment:
We never completely remove older cases. This settings should still remain in the code with a deprecation warning.